### ISSUE TITLE
Document installation of pum / pirogue

### ISSUE THREAD

**Issue by m-kuhn** [QGEP/docs#107]
Especially for Windows

**Comment by m-kuhn** [comment]
@3nids is it currently possible to install pirogue on Windows?

**Comment by TriformDaniel** [comment]
- pum installation seems well documented on https://github.com/opengisch/pum (haven't tried it yet).
- But I haven't found any pirogue installation doc neither on https://github.com/opengisch/pirogue nor https://opengisch.github.io/pirogue/
Problem encountered when reading https://qgep.github.io/docs/installation-guide/requirements.html

**Comment by 3nids** [comment]
@m-kuhn I have not tested it yet. While testing pum on windows, I made some adaptions which might need to be ported to pirogue (the way is build the wheel, shebangs and stuff like this).
This would require further investigations.

**Comment by TriformDaniel** [comment]
> @m-kuhn I have not tested it yet. While testing pum on windows, I made some adaptions which might need to be ported to pirogue (the way is build the wheel, shebangs and stuff like this).
> This would require further investigations.

Does this need to be sorted before I can install QGEP?

**Comment by m-kuhn** [comment]
No, you can safely skip this step, it will only be required for an update of the datamodel in the database in the future.

It's on the todo list for Q1 2020 to fix this on Windows (I assume that's the platform you need), just ignore this part for the moment.

**Comment by TriformDaniel** [comment]
> No, you can safely skip this step, it will only be required for an update of the datamodel in the database in the future.
> 
> It's on the todo list for Q1 2020 to fix this on Windows (I assume that's the platform you need), just ignore this part for the moment.

Ah. Great. Actually we still have the option to install posgreSQL with postGIS on either linux (ubuntu) or Windows. I guess linux would be better, right? QGIS (with QGEP lateron) runs on Windows but could probably also be installed in a linux environment. What's better here?

**Comment by m-kuhn** [comment]
Pirogue/pum does not need to run on the same machine as postgres, so that's not something to take into account for this choice.

You need to choose the platform you (or your service provider) is comfortable maintaining.

**Comment by ponceta** [comment]
https://devblogs.microsoft.com/commandline/announcing-wsl-2/

This looks very promising and could maybe fix these linux/windows issues.

To be followed in the next few months.

**Comment by ponceta** [comment]
> Pirogue/pum does not need to run on the same machine as postgres, so that's not something to take into account for this choice.
> 
> You need to choose the platform you (or your service provider) is comfortable maintaining.

Maybe this should appear in documentation :)

http://qgep.github.io/docs/fr/admin-guide/db-update-pum/index.html#requirements

https://github.com/QGEP/docs/pull/133

**Comment by sjib** [comment]
@TriformMoritz Is this still an issue or can I close that?
